spring-shell-autoconfigure/src/main/java/org/springframework/shell/boot/ParameterResolverAutoConfiguration.java

```diff
@@ -1,24 +1,35 @@
 package org.springframework.shell.boot;
 
-import java.util.Set;
-import java.util.stream.Collectors;
+import java.util.ArrayList;
+import java.util.List;
 
-import org.springframework.beans.factory.ObjectProvider;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.core.convert.ConversionService;
-import org.springframework.shell.ParameterResolver;
-import org.springframework.shell.standard.StandardParameterResolver;
-import org.springframework.shell.standard.ValueProvider;
+import org.springframework.core.convert.support.DefaultConversionService;
+import org.springframework.messaging.handler.annotation.support.HeadersMethodArgumentResolver;
+import org.springframework.messaging.handler.invocation.HandlerMethodArgumentResolver;
+import org.springframework.shell.command.ArgumentHeaderMethodArgumentResolver;
+import org.springframework.shell.command.CommandContextMethodArgumentResolver;
+import org.springframework.shell.command.CommandExecution.CommandExecutionHandlerMethodArgumentResolvers;
+import org.springframework.shell.completion.CompletionResolver;
+import org.springframework.shell.completion.DefaultCompletionResolver;
+import org.springframework.shell.standard.ShellOptionMethodArgumentResolver;
 
 @Configuration(proxyBeanMethods = false)
 public class ParameterResolverAutoConfiguration {
 
 	@Bean
-	public ParameterResolver standardParameterResolver(ConversionService conversionService,
-			ObjectProvider<ValueProvider> valueProviders) {
-		Set<ValueProvider> collect = valueProviders.orderedStream().collect(Collectors.toSet());
-		return new StandardParameterResolver(conversionService, collect);
+	public CompletionResolver defaultCompletionResolver() {
```

Comment:
Any need to disable or override this default?

Reply:
Currently I don't think so.